app/jobs/compute_rankings_records.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class ComputeRankingsRecords < WcaCronjob
+  def self.reason_not_to_run
+    unless ComputeAuxiliaryData.last_run_successful?
```

Comment:
should we not also check if we already computed the table since CAD ran last? Otherwise this looks to me like this will just run every 30 minutes regardless if there are new PR information or not